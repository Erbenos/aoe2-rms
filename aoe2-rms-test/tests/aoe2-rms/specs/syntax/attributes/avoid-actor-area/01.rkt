#lang aoe2-rms
; Place a barracks for empire wars but have it avoid various other objects that you already placed.

<OBJECTS-GENERATION>
(create-object 'BARRACKS
               (set-place-for-every-player)
               (min-distance-to-players 7)
               (max-distance-to-players 9)
               (avoid-actor-area 94)
               (avoid-actor-area 40)
               (avoid-actor-area 8)
               (avoid-actor-area 9)
               (avoid-actor-area 99)
               (avoid-actor-area 171)
               (actor-area 51)
               (actor-area-radius 5))
