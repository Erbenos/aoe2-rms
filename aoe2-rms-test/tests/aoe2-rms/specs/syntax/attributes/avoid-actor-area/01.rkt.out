<OBJECTS_GENERATION>
create_object BARRACKS {
  set_place_for_every_player
  min_distance_to_players 7
  max_distance_to_players 9
  avoid_actor_area 94
  avoid_actor_area 40
  avoid_actor_area 8
  avoid_actor_area 9
  avoid_actor_area 99
  avoid_actor_area 171
  actor_area 51
  actor_area_radius 5
}
